### ISSUE TITLE
Kuliah 6 Mei 2023

### ISSUE THREAD

**Issue by dudung** [dudung/sk5003-02-2022-2#9]
**Import modul built-in dan eksternal** \
`slide` [10.5281/zenodo.](https://doi.org/10.5281/zenodo.) \
`video` [](https://youtu.be/) \
`tugas` [](https://github.com/dudung/sk5003-02-2022-2/issues/7#issuecomment-1509401291) \
`ednex` https://edunex.itb.ac.id/courses/48923/preview/168407

+ Silakan menyampaikan hal-hal yang diperlukan untuk kuliah hari ini di sini, seperti pertanyaan, diskusi, saran, kritik, dan lain-lain. Terima kasih.

**Comment by Andinikma19** [comment]
Ijin pak, untuk slidenya tidak bsa diakses (DOI not found)

**Comment by aliefpascal** [comment]
<img width="194" alt="image" src="https://user-images.githubusercontent.com/127569001/236591315-15eb6ec9-ed87-40b2-a7d0-f177bdf54fdc.png">

untuk pemberian legendnya itu pasti berurut ya pak, data 1 ke data 2 dan data 3?

makasih pak

**Comment by tororahmanz** [comment]
![image](https://user-images.githubusercontent.com/44026991/236591556-801a81e2-a246-4364-a12b-5668518ecb60.png)

pak apakah untuk syntax plot ini ada batasan , misal kita ingin membuat 10 plot dalam syntax tersebut apakah bisa ?

**Comment by Andinikma19** [comment]
Apakah bisa untuk mengubah warna backgroud statistiknya pak ? trima kasih

**Comment by dudung** [comment]
> Ijin pak, untuk slidenya tidak bsa diakses (DOI not found)

Iya, Bu @Andinikma19. Baru saja saya unggah, juga dengan videonya. Terima kasih.
